code/particle/util/ParticleProperties.cpp

```diff
@@ -34,8 +34,28 @@ void ParticleProperties::parse(bool nocreate) {
 	}
 }
 
+int ParticleProperties::chooseBitmap()
+{
+	// failsafe check, though we should not have been able to get to this point
+	if (m_bitmap_list.empty())
```

Comment:
Since returning `-1` here will cause issues later down the line (since bitmap particles can not be rendered without a texture) it would be better to just use an `Assertion` here to check that the list is not empty.